pkg/tfbridge/schema.go

```diff
@@ -718,7 +724,12 @@ func (ctx *conversionContext) applyDefaults(result map[string]interface{}, olds,
 			}
 			if defaultValue != nil {
 				glog.V(9).Infof("Created Terraform input: %v = %v (from %s)", name, defaultValue, source)
-				result[name] = defaultValue
+				pValue := resource.NewPropertyValue(defaultValue)
```

Comment:
Well this is a massive transformation introduced here! We need to be careful. We're losing track of what the actual type is of defaultValue because it all collapses to any. There's two options it's either "any" representation of pulumi.PropertyValue or it's the "any" representation of TF value (hcty.Value).

Reply:
Your right. This transformation should have been applied earlier. Fixed in https://github.com/pulumi/pulumi-terraform-bridge/pull/1322/commits/796df14c8400440358b5b2075cf16340c2f782b7.

Reply:
Eh! I'm not sure! That depends on what we expect users to put in defaultValue. Just need to be careful if we're changing it need to call it out... change management..